subsys/bluetooth/host/hci_core.c

```diff
@@ -802,6 +834,13 @@ static void le_conn_complete(struct net_buf *buf)
 		}
 	}
 
+	if (BT_FEAT_LE_DLE(bt_dev.le.features)) {
```

Comment:
How come you're doing this both here and in le_remote_feat_complete? Wouldn't you want to wait until you know if the remote supports the feature or not?